ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GhcrIntegrationForm.tsx

```diff
@@ -0,0 +1,326 @@
+                            id="config.docker.username"
+                            value={values.config.docker.username}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable}
+                        />
+                    </FormLabelGroup>
+                    {!isCreating && isEditable && (
+                        <FormLabelGroup
+                            fieldId="updatePassword"
+                            helperText="Enable this option to replace currently stored credentials (if any)"
+                            errors={errors}
+                        >
+                            <Checkbox
+                                label="Update stored credentials"
+                                id="updatePassword"
+                                isChecked={values.updatePassword}
+                                onChange={(event, value) => onUpdateCredentialsChange(value, event)}
+                                onBlur={handleBlur}
+                                isDisabled={!isEditable}
+                            />
+                        </FormLabelGroup>
+                    )}
+                    <FormLabelGroup
+                        label="Password"
+                        labelIcon={
+                            <IntegrationHelpIcon
+                                helpTitle="GitHub password"
+                                helpText={
+                                    <Flex direction={{ default: 'column' }}>
+                                        <FlexItem>
+                                            A GitHub personal access token with at least{' '}
+                                            <em>read:packages</em> scope. For more information, see{' '}
+                                            <ExternalLink>
+                                                <a
+                                                    href="https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry"
+                                                    target="_blank"
+                                                    rel="noopener noreferrer"
+                                                >
+                                                    Authenticating to the Container registry
+                                                </a>
+                                            </ExternalLink>
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for password"
+                            />
+                        }
+                        fieldId="config.docker.password"
+                        touched={touched}
+                        errors={errors}
+                    >
```

Comment:
Currently it appears a password is required to create a GHCR integration via the UI (the `Save` button is disabled otherwise). If this is intentional we add the "red asterisk" to indicate the field is required.

Before doing so however refer to my other comment, I don't believe password should be required.